node/node.go

```diff
@@ -210,6 +210,7 @@ func (n *Node) Start() error {
 		for _, s := range services {
 			// TODO-Klaytn-ServiceChain call setcomponents repeatedly for same component
 			s.SetComponents(service.Components())
+			s.SetComponents([]interface{}{service})
```

Comment:
You added this line because Components() of serviceChain is nil, right?

Reply:
Uhm, no. I added it to leverage CN features. The first leveraged feature here is to call the `Progress()` function.

Reply:
I erased this line, and no error detected in this line `if bm.subBridge.cn.APIBackend.Progress().HighestBlock > bm.subBridge.blockchain.CurrentBlock().Number().Uint64()`.
So I'm curious this line is really needed.

Reply:
A panic would appear If you remove the code because the `cn` is nil. Could you explain how did you test it?